lib/msgpack/time.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# MessagePack extention packer and unpacker for built-in Time class
+module MessagePack
+  module Time
+    # 3-arg Time.at is available Ruby >= 2.5
+    TIME_AT_3_AVAILABLE = begin
+      !!::Time.at(0, 0, :nanosecond)
+                          rescue ArgumentError
+                            false
+    end
```

Comment:
FYI the above crazy indentation is made by rubocop.

Reply:
How about this code?
```ruby
TIME_AT_3_AVAILABLE = !!::Time.at(0, 0, :nanosecond) rescue false
```

Reply:
rubocop fixes it to:

```
    TIME_AT_3_AVAILABLE = begin
                            !!::Time.at(0, 0, :nanosecond)
                          rescue StandardError
                            false
                          end
```

but yes, it's better than the current, so apply it f49a8ff

I wonder the rubocop behavior. It sometimes seems to change the code at random.